scripts/find_cloudfront_distribution_id.sh

```
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -euo pipefail
+
+domain="$1"
+found_id=""
+
+aliases=$(aws cloudfront get-distribution-config --id "$id" --query "DistributionConfig.Aliases.Items[]" --output text)
```

Shellcheck (0.10.0)

[warning] id is referenced but not assigned.

(SC2154)